packages/chat/src/utils/messageUtil.ts

```diff
@@ -45,3 +45,27 @@ const twitchMessageTypes = new Map<string, MessageConstructor>([
 export function parseTwitchMessage(rawLine: string): Message {
 	return parseMessage(rawLine, undefined, twitchMessageTypes, true);
 }
+
+export function autoSplitterWithSpaces(text: string, maxMsgLength: number): string[] {
+	if (text.length <= maxMsgLength) return [text];
+	text = text.trim();
+	const res = [];
+
+	let startIndex = 0;
+	let endIndex = maxMsgLength;
+
+	while (startIndex < text.length) {
+		let spaceIndex = text.lastIndexOf(' ', endIndex);
+
+		if (spaceIndex === -1 || spaceIndex <= startIndex || text.length - startIndex + 1 <= maxMsgLength)
```

Comment:
Apparently I should really update my eslint rules - I always assumed prettier does that, but apparently it does not...

Anyway, please always use curly braces with conditionals.